website/docs/intro/install/install-opentofu.sh

```diff
@@ -637,6 +637,16 @@ install_rpm() {
   fi
 }
 
+# Function to install OpenTofu on FreeBSD using pkg
```

Comment:
This script should not be in this folder, it got committed by accident. Please remove the changes from the commit, they belong in the `get.opentofu.org` repository.

Reply:
Should i update this file https://github.com/opentofu/get.opentofu.org/blob/main/src/install-opentofu.sh ?

Reply:
I got it, standalone mode worked for me, We don't need any changes.

Reply:
Maybe we should change it to link to `/docs/intro/install/standalone/` instead of mirroring it? We could change the page title to "Standalone (Linux/MacOS/Windows/BSD)"?